examples/jsm/nodes/core/Node.js

```diff
@@ -36,42 +36,19 @@ class Node {
 
 	* getChildren() {
 
-		for ( const property in this ) {
+		const self = this;
 
-			const object = this[ property ];
+		for ( const { prop, childNode } of getNodeChildren( this ) ) {
```

Comment:
I think better use an individual property to indexes like:
```js
// prop to property, added index
const { property, index, childNode } of getNodeChildren( this );

// instead of prop.includes( '/' ) 
if ( index !== null ) {
```

Reply:
I think it's simpler... To manage all three situations: simple property, property and index, two properties.

Reply:
It because one is a property and the other is an index of the property if it exists, it would be the correct logic, two properties. 

Other problems are optimization, we are needing to slice the string every time we need to access the index, and can bring others problems if the name includes `/`.

Reply:
OK, agreed!